src/ray/core_worker/test/core_worker_test.cc

```diff
@@ -75,6 +75,9 @@ std::string MetadataToString(std::shared_ptr<RayObject> obj) {
 class CoreWorkerTest : public ::testing::Test {
  public:
   CoreWorkerTest(int num_nodes) : gcs_options_("127.0.0.1", 6379, "") {
+#ifdef _WIN32
+    RAY_CHECK(!"port system() calls to Windows before running this test");
```

Comment:
Please do `RAY_CHECK(false) << "port system() calls to Windows before running this test"` here, same below.